multisig-wallet/project/multisig-contract/tests/functions/balance.rs

```diff
@@ -0,0 +1,39 @@
+use crate::utils::{
+    abi_calls::balance,
+    test_helpers::{base_asset_contract_id, setup_env, DEFAULT_TRANSFER_AMOUNT},
+    VALID_SIGNER_PK,
+};
+
+use fuels::prelude::*;
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn gets_balance() {
+        let (_private_key, deployer, _non_owner) = setup_env(VALID_SIGNER_PK).await.unwrap();
+
+        let initial_balance = balance(&deployer.contract, base_asset_contract_id())
+            .await
+            .value;
+
+        deployer
+            .wallet
+            .force_transfer_to_contract(
+                deployer.contract.get_contract_id(),
+                DEFAULT_TRANSFER_AMOUNT,
+                BASE_ASSET_ID,
+                TxParameters::default(),
+            )
+            .await
+            .unwrap();
+
+        let final_balance = balance(&deployer.contract, base_asset_contract_id())
+            .await
+            .value;
+
+        assert_eq!(initial_balance, 0);
+        assert_eq!(final_balance, DEFAULT_TRANSFER_AMOUNT);
```

Comment:
Looking at the imported constant we can see that the default is non-zero; however, this test itself does not guarantee that there is any change in the balance before and after because it does not assert that the initial balance and final balance are not the same. 1 more assertion should be added to check this because someone may change the constant in the other file and this test would still pass.